deploy/kubernetes/manifest.yaml

```diff
@@ -65,7 +22,9 @@ spec:
       labels:
         app: fsx-csi-node
     spec:
-      serviceAccount: csi-node-sa
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      serviceAccount: fsx-csi-node-sa
```

Comment:
I'm wonder does node service even need a service account if the sa doesn't have any role bindings?

Reply:
Hm yeah I removed it. I guess there is no harm but it's one less object to create.